config-controller/config/crd/bases/config.stackrox.io_securitypolicies.yaml

```diff
@@ -0,0 +1,200 @@
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SecurityPolicySpec defines the desired state of SecurityPolicy
+            properties:
+              categories:
+                items:
+                  type: string
+                type: array
+              criteriaLocked:
+                type: boolean
+              description:
+                type: string
+              disabled:
+                type: boolean
+              enforcementActions:
+                enum:
+                - UNSET_ENFORCEMENT
+                - SCALE_TO_ZERO_ENFORCEMENT
+                - UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT
+                - KILL_POD_ENFORCEMENT
+                - FAIL_BUILD_ENFORCEMENT
+                - FAIL_KUBE_REQUEST_ENFORCEMENT
+                - FAIL_DEPLOYMENT_CREATE_ENFORCEMENT
+                - FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT
+                items:
+                  type: string
+                type: array
+              exclusions:
+                items:
+                  properties:
+                    deployment:
+                      properties:
+                        name:
+                          type: string
+                        scope:
+                          properties:
+                            cluster:
+                              type: string
+                            label:
+                              properties:
+                                key:
+                                  type: string
+                                value:
+                                  type: string
+                              type: object
+                            namespace:
+                              type: string
+                          type: object
+                      type: object
+                    image:
+                      properties:
+                        name:
+                          type: string
+                      type: object
+                    name:
+                      type: string
+                  type: object
+                type: array
+              isDefault:
+                type: boolean
+              lifecycleStages:
+                items:
+                  enum:
+                  - DEPLOY
+                  - BUILD
+                  - RUNTIME
+                  type: string
+                minItems: 1
+                type: array
+              mitreAttackVectors:
+                items:
+                  properties:
+                    tactic:
+                      type: string
+                    techniques:
+                      items:
+                        type: string
+                      type: array
+                  type: object
+                type: array
+              mitreVectorsLocked:
+                type: boolean
+              name:
```

Comment:
To track the discussion in slack, we may need to move this to metadata.

Reply:
b987f70fbd8e229a04e625676249b8c64514c5c3